test/controller/check.your.answers.controller.spec.unit.ts

```diff
@@ -68,13 +72,16 @@ describe("check company tests", () => {
     expect(response.text).toContain("document.pdf");
   });
 
-  it ("should forward to confirmation page on post", async () => {
+  it ("should forward to confirmation page and submit the objection on post", async () => {
     const res = await request(app)
         .post(OBJECTIONS_CHECK_YOUR_ANSWERS)
         .set("referer", "/")
         .set("Cookie", [`${COOKIE_NAME}=123`]);
+
+    expect(mockSubmitObjection).toBeCalledWith(dummySession);
+
     expect(res.status).toEqual(302);
-    expect(res.text).toContain(OBJECTIONS_CONFIRMATION);
+    expect(res.header.location).toEqual(OBJECTIONS_CONFIRMATION);
```

Comment:
We've done this elsewhere. I think it's a more rigorous check than just examining the page text.